documentation/controls.txt

```diff
@@ -0,0 +1,18 @@
+
+Driver:
+  Left joystick: controls speed/going forward
+    - y values
+  right joystick: controls turning
+    - x values
+  triggers: control gear shifting
+
+Operator:
+  right joystick: elevator up and down
+    - y values
+  hold A and B on both controllers to switch into wing_mode
+  wing_mode
+    - triggers will raise the wings
+    - with just operator safety: the bumper lowers the wing
+  not wing_mode
```

Comment:
From Solomon: Left Trigger should be absorb, Right Trigger should be spit. No one trigger/two trigger control. If both are pressed at the same, the intake should do nothing. Let the trigger have variable speed control.

Reply:
We talked to last year's drive team to come up with these controls, not Solomon who won't be driving. At least this case has an obvious advantages over the accidental mistake you get from symmetrical controls.

Reply:
Wait so should I change it or not?

Reply:
I think we should keep this one, and change the others, since we asked drive team and this is the one that they had the stronger opinion about.

Reply:
Right so I did the Thing with the left and right variables but now I can't see where you told me to do that? Was I not supposed to do that? What happened to the reply chain?

Reply:
Also do I need a new pull request

Reply:
No, if you push this PR will add the new stuff.